app/lib/actions/proxy_action.rb

```diff
@@ -66,12 +68,9 @@ def remote_task
 
     def trigger_proxy_task
       suspend do |_suspended_action|
-        # Noop if batch triggering is used, otherwise create and immediately trigger it
-        unless with_batch_triggering?(input[:proxy_version])
-          remote_task = prepare_remote_task
-          remote_task.trigger(proxy_action_name, proxy_input)
-          output[:proxy_task_id] = remote_task.remote_task_id
-        end
+        remote_task = prepare_remote_task
+        remote_task.trigger(proxy_action_name, proxy_input)
```

Comment:
Wouldn't this immediatelly trigger the task even if we wanted to use the batch triggering?